geode-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHashCommandsFunctionExecutor.java

```diff
@@ -102,14 +102,14 @@ public int hstrlen(ByteArrayWrapper key, ByteArrayWrapper field) {
   @Override
   public long hincrby(ByteArrayWrapper key, ByteArrayWrapper field, long increment) {
     return stripedExecute(key,
-        () -> getRedisHash(key, true)
+        () -> getRedisHash(key, false)
```

Comment:
I assume changing "true" to "false" here makes it match the native Redis behavior, right? If so, that should probably be called out in the main PR description.

Reply:
👍